ts/components/screens/BaseHeader.tsx

```diff
@@ -7,12 +7,17 @@ import AppHeader from "../ui/AppHeader";
 
 import { DEFAULT_APPLICATION_NAME } from "../../config";
 
+import variables from "../../theme/variables";
 import GoBackButton from "../GoBackButton";
 import { InstabugButtons } from "../InstabugButtons";
 
 const styles = StyleSheet.create({
   helpButton: {
     padding: 8
+  },
+
+  noLeft: {
+    marginLeft: variables.contentPadding - 12 // content padding - header padding
```

Comment:
is 12 coming from `appHeaderPaddingHorizontal` ?

```suggestion
    marginLeft: variables.contentPadding - variables.appHeaderPaddingHorizontal
```